app/models/user.py

```diff
@@ -57,10 +57,11 @@ class User(UserMixin, db.Model):
     def __init__(self, **kwargs):
         super(User, self).__init__(**kwargs)
         if self.role is None:
```

Comment:
You would think it happens' here `super(User, self).__init__(**kwargs)` but this is calling the constructor of the parent class (UserMixin), and it doesn't look like UserMixin uses those kwargs (https://flask-login.readthedocs.org/en/latest/_modules/flask_login.html#UserMixin)